src/main/js/dygraph-light/package.json

```diff
@@ -65,7 +65,7 @@
   "dependencies": {
     "babel-polyfill": "^6.13.0",
     "dygraphs": "^2.1.0",
-    "icos-cp-backend": "0.4.1",
+    "icos-cp-backend": "0.4.2",
```

Comment:
This update is probably moot, as we are not adding a new value format, but does not harm